### PR TITLE
add skip link to main content button

### LINKED ISSUE #4425: Add skip navigation link to ethereum.org
**Is your feature request related to a problem? Please describe.**

We don't have any way to skip over our navigation links for users using screen readers.

**Describe the solution you'd like**

Add a 'Skip to main content' link 

**Additional context**

- https://webaim.org/techniques/skipnav/
- https://www.youtube.com/watch?v=UnEItq289lU
- Good example of an accessible skiplink - https://www.gov.uk/ (access page, press tab and notice skip link, hit enter to skip to main content)

**Want to contribute?**

We love contributions from the Ethereum community! Please comment on an issue if you're interested in helping out with a PR.

### PR DESCRIPTION
**Pull request by il3ven**
## Description
I have added a banner on top of the page that is displayed when TAB is pressed. The component for this banner is named `SkipTo`. Pressing ENTER on the banner navigates to an element with `id=main-content`.

I have added the `id=main-content` to all the pages and templates I could find. The id has been mostly added to the first element on the page. Usually, it is the `<Page>`.

The banner on the top changes color based on the theme.

## Related Issue
Fixes #4425

### CODE REVIEW

**Comment by minimalsm** [comment]
Hey @il3ven is this ready for review?

If so, please move it out of draft :-)

**Comment by minimalsm** [comment]
@il3ven

**Comment by il3ven** [comment]
> Hey @il3ven is this ready for review?
> 
> If so, please move it out of draft :-)

I marked it as a draft for an early feedback. Anyway, I have now completed the PR now and it is ready for review. Sorry, for the delay. I was working on something else.

**Review by minimalsm — CHANGES_REQUESTED** [review]
Thanks @il3ven, really great start on this.

I think we might be able to make this a little bit cleaner by targeting the `MainContent` in our [Layout.js file](https://github.com/ethereum/ethereum-org-website/blob/eb6e72418cc02bc442c606ad8aacdf21a01d9a7e/src/components/Layout.js#L153) instead of adding ids to every page. What do you think?

**Review by samajammin — CHANGES_REQUESTED** [review]
Awesome - thanks for this PR @il3ven! 

I agree with @minimalsm on this - I think we could add the `id` to an element with in the Layout component to make this update a bit cleaner.

File: src/components/SkipTo.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    margin-left: 0.5rem;
+    color: ${(props) => props.theme.colors.background};
+    text-decoration: none;
+
+    &:focus {
+        position: static;
+    }
+`
+
+const SkipTo = ({ href_id }) => {
```
Comment (pettinarip) [inline src/components/SkipTo.js:21]: For consistency, let's use camelCase for props names: `hrefId`.

**Review by pettinarip — CHANGES_REQUESTED** [review]
Loved this contribution @il3ven. I've added a minor comment.

File: src/components/SkipTo.js (hunk 1 above)
Comment (minimalsm) [inline src/components/SkipTo.js:21]: Calling this component a SkipLink makes more semantic sense to me

```suggestion
const SkipLink = ({ href_id }) => {
```

**Comment by il3ven** [comment]
You all are giving me more credit than I deserve. This is just a small PR. 😅 I have incorporated all your suggestions. This PR is ready to be merged.

**Review by pettinarip — APPROVED** [review]
lgtm!

**Review by wackerow — APPROVED** [review]
Looks good 😃  Just made the copy intl-ready. Bringing it in 💪🏻

**Pull request merged**